evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -280,6 +281,19 @@ class DummyEstimator(Estimator):
     expected_parameters = {'Dummy!': {'bar': 'baz'}}
     assert pipeline.parameters == expected_parameters
 
+    X, y = X_y_binary
+    automl = AutoMLSearch(problem_type='binary', objective="Log Loss Binary", max_pipelines=5)
+    automl.search(X, y)
+    best_pipeline = automl.best_pipeline
+    new_pipeline = make_pipeline_from_components(best_pipeline.component_graph, ProblemTypes.BINARY)
+    assert best_pipeline.describe() == new_pipeline.describe()
+    best_pipeline.fit(X, y)
+    best_predictions = best_pipeline.predict(X)
+    new_pipeline.fit(X, y)
+    new_predictions = new_pipeline.predict(X)
+    assert np.array_equal(best_predictions, new_predictions)
+    assert np.array_equal(best_pipeline.feature_importance, new_pipeline.feature_importance)
```

Comment:
Cool that you're comparing the predictions! Can you also compare both static and instance pipeline attributes: `name`, `parameters`, `summary`, `component_graph` (can just compare each element with `isinstance` or something), `describe`